kedro/config/omegaconf_config.py

```diff
@@ -331,14 +331,16 @@ def _get_globals_value(self, variable, default_value):
         value = self["globals"]
         for k in keys:
             value = value.get(k)
-            if not value:
+            if value is None:
                 if default_value:
                     _config_logger.debug(
                         f"Using the default value for the global variable {variable}."
                     )
-                    return default_value
-                msg = f"Globals key '{variable}' not found and no default value provided. "
-                raise InterpolationResolutionError(msg)
+                else:
+                    _config_logger.warning(
+                        f"Globals key '{variable}' might be undefined. Using the default 'None' value."
+                    )
+                return default_value
```

Comment:
Why do we care about the `value` here? It could be anything included `None` or `0`. Shouldn't we check where the `key` is a valid key instead of the value?

Reply:
I've made some changes!